### PR TITLE
Bluetooth: controller: Implement 2 additional VS commands

### PR DESCRIPTION
**Pull request by carlescufi**

### CODE REVIEW

File: subsys/bluetooth/controller/hci/hci.c [hunk 1]
```diff
@@ -1704,6 +1704,27 @@ static void vs_write_bd_addr(struct net_buf *buf, struct net_buf **evt)
...
 }
 
+static void vs_read_build_info(struct net_buf *buf, struct net_buf **evt)
+{
+	struct bt_hci_rp_vs_read_build_info *rp;
+
+#define BUILD_TIMESTAMP " BUILD: " __DATE__ " " __TIME__
+#define HCI_VS_BUILD_INFO "Zephyr OS v" \
+	KERNEL_VERSION_STRING BUILD_TIMESTAMP \
+	" (" CONFIG_BT_CTLR_HCI_VS_BUILD_INFO ")"
```
Comment (holtmann) [inline subsys/bluetooth/controller/hci/hci.c]: No a huge fan that you get "Zephyr OS v9.x.x BUILD: foo bar ()" if the string is not configured. If the BUILD_INFO config entry is "", then we might want to skip also the " ()". Any chance we can make this conditional.

File: subsys/bluetooth/controller/hci/hci.c [hunk 2]
```diff
@@ -1704,6 +1704,27 @@ static void vs_write_bd_addr(struct net_buf *buf, struct net_buf **evt)
...
+
+	const char build_info[] = HCI_VS_BUILD_INFO;
+#define BUILD_INFO_EVT_LEN (sizeof(struct bt_hci_evt_hdr) + \
+			    sizeof(struct bt_hci_evt_cmd_complete) + \
+			    sizeof(struct bt_hci_rp_vs_read_build_info) + \
+			    sizeof(build_info))
+	BUILD_ASSERT(CONFIG_BT_RX_BUF_LEN > BUILD_INFO_EVT_LEN);
```
Comment (jhedberg) [inline subsys/bluetooth/controller/hci/hci.c]: Is > really correct here, shouldn't it be >=?

**Review by jhedberg — CHANGES_REQUESTED** [review]
Looks good. Just one small comment (which I may be wrong about)

File: subsys/bluetooth/controller/hci/hci.c (hunk 2 above)
Comment (carlescufi) [reply subsys/bluetooth/controller/hci/hci.c]: Yep, will fix

File: subsys/bluetooth/controller/hci/hci.c (hunk 1 above)
Comment (carlescufi) [reply subsys/bluetooth/controller/hci/hci.c]: Yep, will do that

**Comment by carlescufi** [comment]
@holtmann @jhedberg comments addressed

**Comment by carlescufi** [comment]
@holtmann @jhedberg  opted for a simple concatenation and documented the lack of a space character

**Review by jhedberg — APPROVED** [review]

**Comment by carlescufi** [comment]
@holtmann @cvinayak if there are no further comments I will merge this soon

**Pull request merged**